docs/aws/README.md

```diff
@@ -15,38 +35,21 @@ KARPENTER_VERSION=$(curl -fsSL \
   | jq -r '.tag_name')
 ```
 
-### Create a Cluster
+## Create a Cluster
 
-Karpenter can run anywhere, including on self-managed node groups, [managed node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html), or [AWS Fargate](https://aws.amazon.com/fargate/).
```

Comment:
I think it's really important that we don't remove this line. I don't want readers to misinterpret this as Karpenter relying on Fargate.